dotnet/Services/ProductReviewService.cs

```diff
@@ -740,15 +783,21 @@ public async Task<ReviewsResponseWrapper> GetReviews(int from, int to)
             return await _productReviewRepository.GetProductReviewsMD(string.Empty, from.ToString(), to.ToString());
         }
 
-        public async Task<bool> DeleteReview(string[] ids)
+        public async Task<(bool, HttpStatusCode)> DeleteReview(string[] ids)
```

Comment:
@btalma Will extending the return type here have any effect on the REST API?

Reply:
It should safe change - currently the REST API returns the entire object, it will just have additional value.

Reply:
This shouldn't be a concern anymore as the services aren't changed; the resolver changed instead